packages/chat-stateful-client/src/ChatClientState.ts

```diff
@@ -37,10 +50,26 @@ export type ChatThreadClientState = {
    * keyed by {@link @azure/communication-chat#ChatParticipant.id}.
    */
   participants: { [key: string]: ChatParticipant };
+  /**
+   * Id of this chat thread. Returned from {@link @azure/communication-chat#ChatThreadClient.threadId}
+   */
   threadId: string;
+  /**
+   * An object includes properties of this chat thread.
```

Comment:
An object `containing` properties...

Reply:
And also link the ChatThreadProperties object here.
{@link @azure/communication-react#ChatThreadProperties}

Reply:
Landing this PR so I can start one on top. Will address this comment in my PR.